gazebo_plugins/src/gazebo_ros_camera_utils.cpp

```diff
@@ -518,22 +539,23 @@ void GazeboRosCameraUtils::Init()
     (static_cast<double>(this->width_)) /
     (2.0 * tan(hfov / 2.0));
 
-  if (this->focal_length_ == 0)
+  if (this->focal_length_x_ == 0 || this->focal_length_y_ == 0)
   {
-    this->focal_length_ = computed_focal_length;
+    this->focal_length_x_ = computed_focal_length;
+    this->focal_length_y_ = computed_focal_length;
   }
   else
   {
     // check against float precision
-    if (!ignition::math::equal(this->focal_length_, computed_focal_length))
+    if (!ignition::math::equal(this->focal_length_x_, computed_focal_length))
```

Comment:
Should `focal_length_y_` also be checked?

Reply:
Good afternoon :)
I added the checking of focal_length_y_ as suggested.